plugins/optimization-detective/load.php

```diff
@@ -107,14 +110,12 @@ static function ( string $version ): void {
 		require_once __DIR__ . '/class-od-strict-url-metric.php';
 		require_once __DIR__ . '/class-od-url-metric-group.php';
 		require_once __DIR__ . '/class-od-url-metric-group-collection.php';
-		class_alias( OD_URL_Metric_Group::class, 'OD_URL_Metrics_Group' ); // Temporary class alias for back-compat after rename.
-		class_alias( OD_URL_Metric_Group_Collection::class, 'OD_URL_Metrics_Group_Collection' ); // Temporary class alias for back-compat after rename.
```

Comment:
I removed these because their removal was long-overdue, as these were deprecated in October 2024.

See also this deprecated accessor which can be removed soon as well:

https://github.com/WordPress/performance/blob/da320f3705a95a379993451d30818c379486aaad/plugins/optimization-detective/class-od-tag-visitor-context.php#L123-L136